lib/hammer_cli_katello/content_view_purge.rb

```diff
@@ -0,0 +1,54 @@
+module HammerCLIKatello
+  class ContentViewPurgeCommand < HammerCLIKatello::Command
+    command_name "purge"
+    desc "Delete old versions of a content view"
+
+    option "--id", "ID", _("content view numeric identifier"), required: true
```

Comment:
For consistency, we may want to enhance this in the future to also support --name, which would mean id resolving with organization/organization-id.  We can consider this outside of the scope of the PR, but may be useful to create a redmine to track.